lib/vssolution.hpp

```diff
@@ -77,7 +77,7 @@ class VisualizationSceneSolution : public VisualizationSceneScalarData
                         Vector &values, int sides, Array<double> &lvl,
                         int flat = 0);
 
-   int GetAutoRefineFactor();
+   virtual int GetFunctionAutoRefineFactor();
```

Comment:
Use `override` instead of `virtual`?

Reply:
I used overrides, but I had to remove them in https://github.com/GLVis/glvis/pull/285/commits/f899b96df4a76d6c487a0835d8a3df2f0c1781a8 , because of `-Winconsistent-missing-override` in CI (see https://github.com/GLVis/glvis/actions/runs/10047196329/job/27768344278). So we can use `virtual` everywhere or `override`, not a mix. Due to the shift in MFEM towards `override` (which I like 😉 ), I will change it at least in these classes to it, but it is a bit beyond the scope of this PR😕 .